featurebyte/worker/task_executor.py

```diff
@@ -180,7 +181,11 @@ async def execute(self) -> Any:
         try:
             # Execute the task
             await self._update_task_start_time_and_description(payload_obj)
-            await self.task.execute(payload_obj)
+            task_result = await self.task.execute(payload_obj)
```

Comment:
If we return the output at the end (and fix `_execute_task` and `execute` in `TaskExecutor` to return the output I think celery should populate the result field. This is fine as well as long we are sure that celery will not overwrite the results

Reply:
would be better to add an integration test since celery also update the result field

Reply:
Updated integration test to cover this & rename the attribute to `task_result`.